api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -58,6 +58,8 @@ public void testCreateAndDeleteTestUser() {
                 });
     // Ensure our two custom schema fields are correctly set & re-fetched from GSuite.
     assertThat(aouMeta).containsEntry("Institution", "All of Us Research Workbench");
+    assertThat(aouMeta)
```

Comment:
This test is disabled because it's very flaky (see [RW-7743](https://precisionmedicineinitiative.atlassian.net/browse/RW-7743)). I was able to get it to pass by temporarily removing the `@Disabled` annotation and removing the `isUsernameTaken` checks. However, it was still flaky. As the flakiness seems unrelated to Absorb, I have not investigated further.